apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -393,376 +344,335 @@ export const PoolAdvertisementPoster = ({
-                              </Text>
-                            </div>
-                          </Accordion.Content>
-                        </Accordion.Item>
-                      ),
-                    )}
-                  </Accordion.Root>
-                </>
-              ) : null}
+              <Heading level="h3" size="h4">
+                {intl.formatMessage({
+                  defaultMessage: "Minimum experience or education",
+                  id: "v6boy9",
+                  description:
+                    "Title for minimum experience or education section of a pool advertisement",
+                })}
+              </Heading>
+              <Text>
+                {intl.formatMessage(
+                  {
+                    defaultMessage:
+                      "{title} roles require a minimum amount of experience or a relevant degree.",
+                    id: "LnISTX",
+                    description:
+                      "Descriptive text about experience or education requirements of a pool advertisement",
+                  },
+                  { title: fullTitle },
+                )}
+              </Text>
+              <EducationRequirements />
+              <Heading level="h3" size="h4">
+                {intl.formatMessage({
+                  defaultMessage: "Skill requirements",
+                  id: "706kTz",
+                  description:
+                    "Title for skill requirements section of a pool advertisement",
+                })}
+              </Heading>
+              <Text>
+                {intl.formatMessage({
+                  defaultMessage:
+                    'All opportunities on this platform require you to use your application to demonstrate a handful of required "occupational" or "technical" skills. Some opportunities will also assess “behavioural” or "soft" skills independently of your application, though you\'re free to add them to your resume.',
+                  id: "6F4HY/",
+                  description:
+                    "Descriptive text about how skills are defined and used for pool advertisements and applications",
+                })}
+              </Text>
+              <Heading level="h4" size="h6">
+                {intl.formatMessage({
+                  defaultMessage: "Required technical skills",
+                  id: "9V8bnL",
+                  description:
+                    "Title for required technical skills section of a pool advertisement",
+                })}
+              </Heading>
+              <Text>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "The following skills are required for this role, but aren't required as a part of your application.",
+                  id: "SiQPEi",
+                  description:
+                    "Descriptive text about how required technical skills are used in the application process",
+                })}
```

Comment:
I do not understand this part, you won't be able to submit an application without the required technical skills so they are required. 

The message appears to be the same as the bit underneath "Required behavioural skills", typo in Figma? As behavioural skills aren't filled in in the application

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/93dc018e-6663-4ec4-af2b-f1680118e532)

Reply:
That is a good question, it does not seem to make much sense but that was what was in the figma 🤷‍♀️ I'll try to remember to ask during standup.

Reply:
Definitely a mistake. Ask Josh for the correct copy for required skills.

Reply:
Got the updated copy and changed in [update copy](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6543/commits/205318329b8dce1892011f38b3846615fe6609fd). Just waiting for translations now 👍